src/Core/Ord.idr

```diff
@@ -44,22 +65,8 @@ Ord Constant where
         tag (Str _) = 10
         tag (Ch _) = 11
         tag (Db _) = 12
-        tag WorldVal = 13
-        tag IntType = 14
-        tag Int8Type = 15
-        tag Int16Type = 16
-        tag Int32Type = 17
-        tag Int64Type = 18
-        tag IntegerType = 19
-        tag Bits8Type = 20
-        tag Bits16Type = 21
-        tag Bits32Type = 22
-        tag Bits64Type = 23
-        tag StringType = 24
-        tag CharType = 25
-        tag DoubleType = 26
-        tag WorldType = 27
-
+        tag (PrT _) = 13
```

Comment:
It may be inconsequential, but you’ve switched the order of worldval compared to any prim.

Reply:
Yes, I thought that this order is arbitrary anyway, so it does not really matter. Do you think it's worth to make worldval to be the least again?

Reply:
I'm fairly uncertain. I doubt it matters, but thought it was worth the note. It's possible that some code somewhere does need world to get sorted to the end of something or other; it is a pretty special type of thing.